tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -33,1554 +32,1573 @@
 pytestmark = pytest.mark.notrackingurimock
 
 
-class TestSqlAlchemyStoreSqlite(unittest.TestCase):
-    def _get_store(self, db_uri=""):
-        return SqlAlchemyStore(db_uri)
-
-    def _setup_db_uri(self):
-        if _TRACKING_URI_ENV_VAR in os.environ:
-            self.temp_dbfile = None
-            self.db_url = os.getenv(_TRACKING_URI_ENV_VAR)
-        else:
-            fd, self.temp_dbfile = tempfile.mkstemp()
-            # Close handle immediately so that we can remove the file later on in Windows
-            os.close(fd)
-            self.db_url = "{}{}".format(DB_URI, self.temp_dbfile)
-
-    def setUp(self):
-        self._setup_db_uri()
-        self.store = self._get_store(self.db_url)
-
-    def get_store(self):
-        return self.store
-
-    def tearDown(self):
-        if self.temp_dbfile:
-            os.remove(self.temp_dbfile)
-        else:
-            with self.store.ManagedSessionMaker() as session:
-                for model in (
-                    SqlModelVersionTag,
-                    SqlRegisteredModelTag,
-                    SqlModelVersion,
-                    SqlRegisteredModel,
-                ):
-                    session.query(model).delete()
-
-    def _rm_maker(self, name, tags=None, description=None):
-        return self.store.create_registered_model(name, tags, description)
+@pytest.fixture
+def db_uri():
+    if _TRACKING_URI_ENV_VAR in os.environ:
+        return os.getenv(_TRACKING_URI_ENV_VAR)
+    else:
+        fd, temp_dbfile = tempfile.mkstemp()
+        # Close handle immediately so that we can remove the file later on in Windows
+        os.close(fd)
+        yield f"{DB_URI}{temp_dbfile}"
+        os.remove(temp_dbfile)
```

Comment:
```suggestion
        return f"sqlite:///{tmp_path / 'mlflow.db'}"
```

Reply:
Thanks for your review! I'll perform the required changes.